utils/text_to_image/stable_diffusion.py

```diff
@@ -8,7 +10,12 @@ def __init__(self):
         self.available_instances = 100000
 
     def get_input(self):
-        return 'a professional photograph of an astronaut riding a triceratops'
+        adjectives = ['professional', 'enormous', 'fun', 'beautiful', 'small', 'ugly']
+        nouns = ['dog', 'cat', 'astronaut', 'person', 'knight', 'horse', 'soldier']
+        actions = ['runs', 'jumps', 'rides a triceratops', 'rides a bike', 'eats a burger', 'washes clothes']
+        adverbs = ['quickly', 'slowly', 'loudly']
+
+        return f'{random.choice(adjectives)} {random.choice(nouns)} {random.choice(actions)} {random.choice(adverbs)}'
```

Comment:
Cool idea. You could also set random seed to make sure we always get the same combinations.